deeprank/features/FullPSSM.py

```diff
@@ -48,12 +45,11 @@ def __init__(self, mol_name=None, pdb_file=None, chain1='A', chain2='B',
         super().__init__("Residue")
 
         self.mol_name = mol_name
-        self.pdb_file = pdb_file
-        self.pssm_path = pssm_path
+        self.pdb_file = mutant.pdb_path
```

Comment:
"mutant.pdb_path" is same as "pdb_file" isn't it?

Reply:
Yes they are, we could of course completely remove the self.pdb_file and self.pssm_path variables.